app/client/components/consent/PrivacySettings.tsx

```diff
@@ -212,10 +280,26 @@ export class PrivacySettings extends Component<{}, State> {
     );
   }
 
-  private buildState(iabVendorList: ParsedIabVendorList): Promise<void> {
+  private buildState(
+    guPurposeList: ParsedGuPurposeList,
```

Comment:
OK I asked about  the naming of `cleanGuPurposeList` above, I'm guessing that was done so we could name this parameter `guPurposeList`.